### PR TITLE
Update multiphase doc

### PR DESCRIPTION
**Pull request by chicm-ms**

### CODE REVIEW

File: docs/en_US/MultiPhase.md [hunk 1]
```diff
@@ -48,6 +70,10 @@ trial_end
 ```
 With this information, the tuner could know which trial is requesting a configuration, and which trial is reporting results. This information provides enough flexibility for your tuner to deal with different trials and different phases. For example, you may want to use the trial_job_id parameter of generate_parameters method to generate hyperparameters for a specific trial job.
 
-Of course, to use your multi-phase tuner, __you should add `multiPhase: true` in your experiment YAML configure file__.
 
-[ENAS tuner](https://github.com/countif/enas_nni/blob/master/nni/examples/tuners/enas/nni_controller_ptb.py) is an example of a multi-phase tuner.
+### Tuners support multi-phase experiments:
+
+[TEP](BuiltinTuner.md#TPE), [Random](BuiltinTuner.md#Random), [Anneal](BuiltinTuner.md#Anneal), [Evolution](BuiltinTuner.md#Evolution), [SMAC](BuiltinTuner.md#SMAC), [NetworkMorphism](BuiltinTuner.md#NetworkMorphism), [MetisTuner](BuiltinTuner.md#MetisTuner), [ENAS tuner](https://github.com/countif/enas_nni/blob/master/nni/examples/tuners/enas/nni_controller_ptb.py).
```
Comment (xuehui1991) [inline docs/en_US/MultiPhase.md]: BOHB and Hyperband?

Comment (QuanluZhang) [reply docs/en_US/MultiPhase.md]: let me work on it today :)

Comment (QuanluZhang) [reply docs/en_US/MultiPhase.md]: the pr that supporting multi-phase on BOHB and Hyperband is merge, please update the doc accordingly.

Comment (chicm-ms) [reply docs/en_US/MultiPhase.md]: Thanks, updated.

File: docs/en_US/AdvancedFeature/MultiPhase.md [hunk 2]
```diff
@@ -48,6 +70,10 @@ trial_end
...
 
-Of course, to use your multi-phase tuner, __you should add `multiPhase: true` in your experiment YAML configure file__.
 
-[ENAS tuner](https://github.com/countif/enas_nni/blob/master/nni/examples/tuners/enas/nni_controller_ptb.py) is an example of a multi-phase tuner.
+### Tuners support multi-phase experiments:
+
+[TPE](../Tuner/BuiltinTuner.md#TPE), [Random](../Tuner/BuiltinTuner.md#Random), [Anneal](../Tuner/BuiltinTuner.md#Anneal), [Evolution](../Tuner/BuiltinTuner.md#Evolution), [SMAC](../Tuner/BuiltinTuner.md#SMAC), [NetworkMorphism](../Tuner/BuiltinTuner.md#NetworkMorphism), [MetisTuner](../Tuner/BuiltinTuner.md#MetisTuner), [BOHB](../Tuner/BuiltinTuner.md#BOHB), [Hyperband](../Tuner/BuiltinTuner.md#Hyperband), [ENAS tuner](https://github.com/countif/enas_nni/blob/master/nni/examples/tuners/enas/nni_controller_ptb.py).
```
Comment (QuanluZhang) [inline docs/en_US/AdvancedFeature/MultiPhase.md]: readthedocs may not support patterns such as #TPE, #Random.

Comment (chicm-ms) [reply docs/en_US/AdvancedFeature/MultiPhase.md]: Thanks, updated.

**Pull request merged**